testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksUnityCompatibility.java

```diff
@@ -0,0 +1,232 @@
+                    "map_col MAP<TIMESTAMP,INT>," +
+                    "struct_col struct<a: LONG, b: String NOT NULL>" +
+                    ") " +
+                    "USING DELTA " +
+                    "LOCATION '%s'", unityCatalogCatalogName, schemaName, tableName, tableLocation));
+            assertThat(
+                    onTrino().executeQuery("SHOW TABLES IN delta." + schemaName))
+                    .containsOnly(row("t1"));
+            assertThat(
+                    onTrino().executeQuery(format("SHOW COLUMNS IN delta.%s.%s", schemaName, tableName)))
+                    .containsOnly(
+                            row("int_col", "integer", "", ""),
+                            row("string_col", "varchar", "", ""),
+                            row("tinyint_col", "tinyint", "", ""),
+                            row("smallint_col", "smallint", "", ""),
+                            row("bigint_col", "bigint", "", ""),
+                            row("decimal_col", "decimal(10,0)", "", ""),
+                            row("float_col", "real", "", ""),
+                            row("double_col", "double", "", ""),
+                            row("date_col", "date", "", ""),
+                            row("timestamp_col", "timestamp(3) with time zone", "", ""),
+                            row("binary_col", "varbinary", "", ""),
+                            row("bool_col", "boolean", "", ""),
+                            row("array_int_col", "array(integer)", "", ""),
+                            row("map_col", "map(timestamp(3) with time zone, integer)", "", ""),
+                            row("struct_col", "row(a bigint, b varchar)", "", ""));
+        }
+        finally {
+            onDelta().executeQuery(format("DROP SCHEMA IF EXISTS %s.%s CASCADE", unityCatalogCatalogName, schemaName));
+        }
+    }
+
+    @Test(groups = {DATABRICKS_UNITY_HTTP_HMS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testDdl()
+    {
+        String schemaName = "testDdl_" + randomNameSuffix();
+        String tableName = "t1";
+        String schemaLocation = getLocationForSchema(bucketName, externalLocationPath, schemaName);
+        String tableLocation = getLocationForTable(bucketName, externalLocationPath, schemaName, tableName);
+        String trinoLocation = getLocationForSchema(bucketName, externalLocationPath, "trino_test_" + randomNameSuffix());
+        try {
+            onDelta().executeQuery(format("CREATE SCHEMA %s.%s MANAGED LOCATION '%s'",
+                    unityCatalogCatalogName, schemaName, schemaLocation));
+            onDelta().executeQuery(
+                    format("CREATE TABLE %s.%s.%s (c1 int, c2 string) using delta location '%s'",
+                            unityCatalogCatalogName, schemaName, tableName, tableLocation));
+            assertThatThrownBy(
```

Comment:
CREATE SCHEMA on trino should be tested as well.

Reply:
I have it commented out for now until we fix it on the Databricks side. Can we add those in a followup PR?